clang/include/clang/Basic/AttrDocs.td

```diff
@@ -2835,8 +2835,8 @@ def SYCLIntelFPGAInitiationIntervalAttrDocs : Documentation {
 This attribute applies to a loop or a function. Indicates that the loop or
 function should be pipelined with an initiation interval of N. N must be a
 positive integer. Cannot be applied multiple times to the same loop or function.
-Cannot be used on the same loop or function in conjunction with
-disable_loop_pipelining.
+Cannot be used on the same loop or function, or in conjunction with
```

Comment:
These are good changes, but they're unrelated to the patch (split them into an NFC change?).

Reply:
Yes, Done.